packages/sdk/src/utils/discovery.ts

```diff
@@ -16,10 +20,20 @@ const DEFAULT_NODE_REQUIREMENTS = {
 export function defaultPeerDiscoveries(
   pubsubTopics: PubsubTopic[]
 ): ((components: Libp2pComponents) => PeerDiscovery)[] {
-  const discoveries = [
-    wakuDnsDiscovery([enrTree["SANDBOX"]], DEFAULT_NODE_REQUIREMENTS),
+  const discoveries = [];
+  discoveries.push(
     wakuLocalPeerCacheDiscovery(),
     wakuPeerExchangeDiscovery(pubsubTopics)
-  ];
+  );
+
+  //TODO: Add support for The Waku Network
+  //https://github.com/waku-org/nwaku/issues/2593
```

Comment:
I wouldn't add it here as this is not something actionable on our side.
And to prevent another commit in future I would add `TWN` related fleet as fallback as it was in https://github.com/waku-org/js-waku/pull/2015 (but as I see it got erased 😑) In both cases js-waku won't be functioning for some time but at least we won't need to do anything once `RLN` as a service is enabled.

Do I miss anything?